cyral/provider.go

```diff
@@ -171,6 +170,7 @@ func Provider() *schema.Provider {
 			"cyral_repository_datamap":               resourceRepositoryDatamap(),
 			"cyral_repository_identity_map":          resourceRepositoryIdentityMap(""),
 			"cyral_repository_local_account":         resourceRepositoryLocalAccount(),
+			"cyral_repository_user_account":          resourceRepositoryUserAccount(),
```

Comment:
I agree with @VictorGFM here.

Reply:
I think we should stick to the API, in this sense `cyral_repository_user_account` is more appropriate. Please make sure our public docs related to `Database Accounts` will refer to this resource. Also, in the resource documentation in the provider itself, just add an observation that this relates to `database accounts` in order to allow those users that landed on the provider to locate themselves on what this resource looks like in the UI.

Reply:
Awesome, I will keep it as User Accounts. Wilson also agrees, for the following reason: 

> Historically our UI has some influences that go well beyond the engineering team, meaning that it can change over time. If something is called Database account today it could be changed to something else in the future due to some UI change for some specific new workflow/feature/other requirement and the API could remain untouched.